spotify_project/libraries/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class Library(models.Model):
```

Comment:
Looks like this PR was branched of another PR. Generally, once the base PR is merged into master, you ought to merge this PR with master. It'll make all the previous PR stuff drop out